libraries/c_sdk/standard/mqtt/src/private/iot_mqtt_internal.h

```diff
@@ -257,6 +257,15 @@
     #define MAX_NO_OF_MQTT_CONNECTIONS    ( 2 )
 #endif
 
+/**
+ * @brief Default config for Maximum Number of MQTT Subscriptions.
+ * This config can be specified by the application based on number of MQTT
+ * subscriptions.
```

Comment:
Please mention that the maximum number of subscriptions is 2^7 in the shim.

Reply:
Addressed